example_buffer.py

```diff
@@ -68,20 +68,18 @@ def parallel_fill(self, games, threads=8):
         """ games is a list of .tfrecord.zz game records. """
         games.sort(key=os.path.basename)
         # A couple extra in case parsing fails
-        max_games = self.max_size // self.samples_per_game + 10
+        max_games = (self.max_size // self.samples_per_game) + 480
```

Comment:
why 480?

Reply:
The consequences of not having enough positions was worse before #329